EarnerStats.sh

```diff
@@ -0,0 +1,26 @@
+# Makes a file called GenderyearsExperience.csv that containes the gender and yearsExperience of each earner
```

Comment:
1.	Good sorting but you don’t provide only the unique combinations as the question asks. Add uniq to the code for this part and it would be perfect 
a.	0.9
2.	1
3.	Code works but you’re right the answer seems wrong. The issue is with your call to cut after you cat wages.csv. You tell cut to keep the 2 and 4 fields which are the years experience and wages column. Instead you should be keeping 3 and 4 fields then your pipeline gives the correct answer.
a.	0.7

2.6/3